SimG4CMS/Forward/src/ZdcNumberingScheme.cc

```diff
@@ -107,16 +107,16 @@ unsigned int ZdcNumberingScheme::getUnitID(const G4Step* aStep) const {
 
     edm::LogVerbatim("ForwardSim") << "ZdcNumberingScheme:"
                                    << "  getUnitID - # of levels = " << level;
-    for (int ich = 0; ich < level; ich++)
-      edm::LogVerbatim("ForwardSim") << "  " << ich << ": copyno " << copyno[ich] << " name=" << name[ich]
-                                     << "  section " << section << " zside " << zside << " layer " << layer << " fiber "
-                                     << fiber << " channel " << channel << "packedIndex =" << intindex
-                                     << " detId raw: " << std::hex << index << std::dec;
+    for (int ich = 0; ich < level; ich++) {
```

Comment:
This loop is also not needed, LogVerbatim may be moved after the line 45

Reply:
Will do